mobile/lib/src/app.dart

```diff
@@ -1,13 +1,19 @@
 import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:polydodo/src/databse.dart';
```

Comment:
typo and could be relative import

```suggestion
import 'database.dart';
```